### DISCUSSION TITLE
Autodiscover mode but execute only in repos with config file?

### DISCUSSION THREAD

**Discussion by dennislapchenko** [renovatebot/renovate#14792] [Request Help]
Would like to activate `autodiscover` but to only run oin repos where a config file (say `renovate.json`) is present. Not sure this is possible by reading through the docs. Can someone advise please?

**Comment by rarkins** [comment]
I think if you set onboarding=false and requireConfig=true then you'd achieve what you describe. I.e. Renovate will never create an onboarding PR, and it will skip any repo without a config. Therefore it runs only on ones which have one.

**Reply by dennislapchenko** [reply]
[Here](https://docs.renovatebot.com/self-hosted-configuration/#onboarding) it says this:
```
Set this to false only if all three statements are true:
- You've configured Renovate entirely on the bot side (e.g. empty renovate.json in repositories) (!!!)
- You want to run Renovate on every repository the bot has access to
- You want to skip all onboarding PRs
```

I have tried `onboarding=false and requireConfig=true` and renovate runs on all repos it can reach :(

**Reply by rarkins** [reply]
Yes, it might run on them but then it will quit out immediately once it sees they don't have a config file.
